src/Symfony/Contracts/HttpClient/Test/TestHttpServer.php

```diff
@@ -21,7 +21,7 @@ class TestHttpServer
     /**
      * @param string|null $workingDirectory
      */
-    public static function start(int $port = 8057/* , string $workingDirectory = null */): Process
+    public static function start(int $port = 8057/* , ?string $workingDirectory = null */): Process
```

Comment:
```suggestion
    public static function start(int $port = 8057/* , string|null $workingDirectory = null */): Process
```

Reply:
@derrabus But how do you deal with mixed types like `float|int|null`? This code style does not support multiple nullable types.